pkg/gather/clusterconfig/sap_datahubs.go

```diff
@@ -0,0 +1,54 @@
+package clusterconfig
+
+import (
+	"context"
+	"fmt"
+
+	"k8s.io/apimachinery/pkg/api/errors"
+	metav1 "k8s.io/apimachinery/pkg/apis/meta/v1"
+	"k8s.io/apimachinery/pkg/runtime/schema"
+	"k8s.io/client-go/dynamic"
+
+	"github.com/openshift/insights-operator/pkg/record"
+)
+
+// GatherSAPDatahubs collects `datahubs.installers.datahub.sap.com` resources from SAP/SDI clusters.
+//
+// Location in archive: customresources/installers.datahub.sap.com/datahubs/<namespace>/<name>.json
+func GatherSAPDatahubs(g *Gatherer, c chan<- gatherResult) {
+	gatherDynamicClient, err := dynamic.NewForConfig(g.gatherKubeConfig)
+	if err != nil {
+		c <- gatherResult{errors: []error{err}}
+		return
+	}
+
+	records, errors := gatherSAPDatahubs(g.ctx, gatherDynamicClient)
+	c <- gatherResult{records: records, errors: errors}
+}
+
+func gatherSAPDatahubs(ctx context.Context, dynamicClient dynamic.Interface) ([]record.Record, []error) {
+	datahubsResource := schema.GroupVersionResource{Group: "installers.datahub.sap.com", Version: "v1alpha1", Resource: "datahubs"}
```

Comment:
We already have a variable for this GVR in 4.7. Can you please update it?

Reply:
I guess it was just now added with the changes that I rebased on top of because it wasn't there when I created the PR.

Reply:
Your guess is correct AFAIK :)